rust/extractor/src/main.rs

```diff
@@ -163,21 +161,25 @@ impl<'a> Extractor<'a> {
         file: &Path,
         semantics: &Semantics<'_, RootDatabase>,
         vfs: &Vfs,
-    ) -> Result<(), String> {
+    ) -> Result<(), RustAnalyzerNoSemantics> {
         let before = Instant::now();
         let Some(id) = path_to_file_id(file, vfs) else {
-            return Err("not included in files loaded from manifest".to_string());
+            return Err(RustAnalyzerNoSemantics::warning(
+                "not included in files loaded from manifest",
+            ));
         };
         match semantics.file_to_module_def(id) {
-            None => return Err("not included as a module".to_string()),
+            None => {
+                return Err(RustAnalyzerNoSemantics::info("not included as a module"));
```

Comment:
So if I'm understanding correctly, this (and the one immediately below) are the bits of this change that affect results - downgrading these warnings to `info` so that they're no longer output by `rust/diagnostics/extraction-warnings` (and thus, we believe, won't appear on the status page).